### PR TITLE
configure the fabric8 maven plugin to use the Red Hat base Java image by default

### PR DESCRIPTION
**Pull request by Ladicek**
Resolves https://github.com/openshiftio/appdev-planning/issues/36

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -56,6 +56,8 @@
     <maven-war-plugin.version>3.0.0</maven-war-plugin.version>
 
     <booster-common.version>2</booster-common.version>
+
+    <fabric8.generator.from>registry.access.redhat.com/redhat-openjdk-18/openjdk18-openshift</fabric8.generator.from>
```
Comment (cescoffier) [inline pom.xml:60]: Is this registry available when not running behind VPN?

Comment (cescoffier) [reply pom.xml:60]: Should it be in a separate profile or be the default?

Comment (Ladicek) [reply pom.xml:60]: This is the Red Hat public Docker registry. Feel free to try :-)

And I think we want to use it by default, per https://github.com/openshiftio/appdev-planning/issues/36

Comment (cescoffier) [reply pom.xml:60]: Ok :-)

**Comment by cescoffier** [comment]
Thanks, scheduled in the release v8.

**Pull request merged**